internal/wclayer/wclayer.go

```diff
@@ -1,5 +1,9 @@
 package wclayer
 
+// This package provides bindings to HCS's legacy layer management API and
```

Comment:
Move this comment up so it's annotating `package wclayer`, and rephrase it as `Package wclayer provides...`.

Reply:
Done